src/sql/parts/modelComponents/modelViewContent.component.ts

```diff
@@ -71,7 +71,8 @@ export class ModelViewContent extends ViewBase implements OnInit, IModelView {
 
 	@memoize
 	public get connection(): sqlops.connection.Connection {
-		if (!this._commonService.connectionManagementService) {
+		if (!this._commonService.connectionManagementService
+			|| !this._commonService.connectionManagementService.connectionInfo) {
```

Comment:
I already made this change and the below in master which is probably why you have a merge conflict